src/chains/mainnet/odin.json

```diff
@@ -0,0 +1,18 @@
+{
+  "chain_name": "odin-mainnet-freya",
```

Comment:
Do you mind if we change chain_name to `odin` or `odin-mainnet`. 
It don't need to set chain id here.